src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAccountManager.java

```diff
@@ -60,17 +57,12 @@
 
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.ws.rs.core.Response;
 import java.io.IOException;
 import java.net.URI;
 import java.security.NoSuchAlgorithmException;
 import java.security.spec.InvalidKeySpecException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```

Comment:
I've actually set it to allow `*` imports if there are more than 15 imports, because otherwise it seems crazy. But it is best to be explicit.
I allow import `*` blanket for `uk.ac.cam.cl.dtg.segue.api.Constants` and `uk.ac.cam.cl.dtg.isaac.api.Constants` which is an option in IntelliJ.

Reply:
Probably just my PTSD from ticking Further Java then.

Reply:
Thanks, updated to replace these with explicit imports.